src/main/java/bio/terra/app/controller/converters/EnumerateSortByParamConverter.java

```diff
@@ -0,0 +1,28 @@
+package bio.terra.app.controller.converters;
+
+import bio.terra.app.controller.exception.ValidationException;
+import bio.terra.model.EnumerateSortByParam;
+
+import org.springframework.core.convert.converter.Converter;
+import org.springframework.stereotype.Component;
+
+import java.util.Arrays;
+import java.util.Collections;
+import java.util.stream.Collectors;
+
+@Component
+public class EnumerateSortByParamConverter
+    implements Converter<String, EnumerateSortByParam> {
+
+    @Override
+    public EnumerateSortByParam convert(String source) {
+        EnumerateSortByParam result = EnumerateSortByParam.fromValue(source.toLowerCase());
+        if (result == null) {
+            String error = String.format("sort must be one of: (%s).",
+                Arrays.stream(EnumerateSortByParam.values())
+                    .map(EnumerateSortByParam::toString)
+                    .collect(Collectors.joining(", ")));
```

Comment:
Using `Arrays.toString()` should work here
```suggestion
                Arrays.toString(EnumerateSortByParam.values()));
```

Reply:
This will require tweaking the expected result in the tests, since it drops some enclosing brackets. I think its a worthwhile change though, so I'll make it happen.